src/Classes/Cli/TextRenderer.php

```diff
@@ -24,37 +24,75 @@ public static function color(string $text, int $color): string
         return "\e[" . $color . "m" . $text . "\e[0m";
     }
 
-    public static function rightPad(string $text, int $totalLength): string
+    public static function rightPad(string $text, int $length): string
     {
-        $textLength = strlen($text);
+        return \str_pad($text, $length, ' ', \STR_PAD_RIGHT);
+    }
 
-        if ($textLength >= $totalLength) {
-            return $text;
-        }
+    public static function getMaxLength(array $items): int
+    {
+        $maxLength = 0;
 
-        $paddingLength = $totalLength - $textLength;
-        $padding = str_repeat(' ', $paddingLength);
+        foreach ($items as $item) {
+            $currentLength = \mb_strlen($item);
+            $maxLength = \max($maxLength, $currentLength);
+        }
 
-        return $text . $padding;
+        return $maxLength;
     }
 
+    /**
+     * @deprecated 1.21.0 Use a `HelpRenderer` instead.
+     *
+     * @param string $heading
+     *
+     * @return string
+     */
     public static function renderHelpHeading(string $heading): string
     {
         return self::color($heading, self::COLOR_YELLOW) . "\n";
     }
 
+    /**
+     * @deprecated 1.21.0 Use a `HelpRenderer` instead.
```

Comment:
👍 I like the idea that you marked that as deprecated.

Reply:
I love how Intelliphense will highlight it then
![image](https://github.com/RobinTheHood/ModifiedModuleLoaderClient/assets/45571053/6ad4ed87-e66a-4da0-9f79-e8193a74fe4e)